airbyte-workers/src/main/java/io/airbyte/workers/general/DefaultReplicationWorker.java

```diff
@@ -128,6 +128,12 @@ public final ReplicationOutput run(final StandardSyncInput syncInput, final Path
     destinationConfig.setCatalog(mapper.mapCatalog(destinationConfig.getCatalog()));
 
     final long startTime = System.currentTimeMillis();
+    final long replicationStartTime = startTime;
```

Comment:
since we already have start time stored on the sync output, do we need to also store in on SyncStats? sorry, I think what I said before missed this, but I don't think we need to track this twice

Reply:
There is a nuance... If the platform is slow to start the source container, startTime might not be the same as replicationStartTime or sourceReadStartTime.

* startTime is the start of the workflow
* replicationStartTime is the start time of the activity
* sourceReadStartTime is when the source container booted.

There might be delays in every step of the process (downloading the docker image, starting the K8s pod, etc)

Reply:
... I'll add these notes to the description!

Reply:
Added!